### ISSUE TITLE
Should assistive tech support navigation to hidden descriptions?

### ISSUE THREAD

**Issue by AmeliaBR** [w3c/core-aam#3]
The [CORE-AAM section on Relations](https://w3c.github.io/core-aam/#mapping_additional_relations_reverse_relations) requires that assistive technologies SHOULD provide a method to navigate to (and within) structured content referenced by `aria-describedby`.

> Note that aria-describedby may reference structured or interactive information where users would want to be able to navigate to different sections of content. User agents MAY provide a way for the user to navigate to structured information referenced by aria-describedby and assistive technology SHOULD provide such a method.

What happens when the element referenced by `aria-describedby` is hidden? In that case, the user agent is supposed to compile the hidden element's plain text as part of the referencing element's accessible description, but the hidden element is not directly included in the accessibility tree. So there is nothing for assistive tech to navigate to.

Is that acceptable? If so, the quoted passage above should be qualified to say "if the referenced element is not hidden" or "if the referenced element is included in the accessibility tree".

If we instead want to support access to structured descriptions even if they are hidden, then the rules for including elements in the accessibility tree need to change.

(I'm converting SVG-AAM spec issues into GitHub issues, and this was one of them, because SVG `<desc>` is a hidden description that we don't currently include in the accessibility tree.)

**Comment by devarshipant** [comment]
> If so, the quoted passage above should be qualified to say "if the referenced element is not hidden" or "if the referenced element is included in the accessibility tree".

That is a good point. I like the wording in "if the referenced element is not hidden."

Also, the word "navigate" can be misleading when using aria-describedby. It allows users to hear the describedby reference text. In this [example ](https://codepen.io/devpant/pen/zMPKLP) both JAWS and NVDA speak the describedby reference. There is no navigation as is generally perceived.
